deploy/olm-catalog/kogito-operator/manifests/kogito-operator.clusterserviceversion.yaml

```diff
@@ -1018,5 +1018,5 @@ spec:
   maturity: alpha
   provider:
     name: Red Hat
-  replaces: kogito-operator.v0.13.0
-  version: 1.0.0-snapshot
+  replaces: kogito-operator.v1.0.0-snapshot
```

Comment:
Shouldn't be here v0.14.0?

Reply:
it should... let me update the PR

Reply:
updated. there is an issue opened to update that part correctly.